.github/workflows/main.yml

```diff
@@ -274,7 +274,7 @@ jobs:
         working-directory: out
 
       - name: Login to Tailscale
-        uses: tailscale/github-action@v3
+        uses: tailscale/github-action@5d536117a7378beffcd6d8523ce73c50fa98684d # v3.2.1
```

Comment:
Please consider Tailscale to be a trusted party and pin to a tag instead of SHA.